### PR TITLE
feat: Replace the Figerona font with Poppins

### PR DESCRIPTION
**Pull request by mimi89999**
### Related Issue

https://github.com/Pool-Of-Tears/Myne/issues/208

## Type of change 

feature

### Pull Request checklist

- [x] The commit message uses the [conventional commiting method][conv-commits].
- [x] Made sure that your PR is not duplicate
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not (for bug fixes/features). Your PR should pass all CI checks in our Gtihub Actions [Workflow](https://github.com/Pool-Of-Tears/Myne/actions)
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (*optional*)

[conv-commits]:https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index

### CODE REVIEW

File: app/src/main/java/com/starry/myne/ui/theme/Type.kt
Comment (starry-shivam) [inline app/src/main/java/com/starry/myne/ui/theme/Type.kt:79]: Please remove these additional italic fonts, as they aren't used anywhere and will just increase the size of the APK file.

**Review by starry-shivam — CHANGES_REQUESTED** [review]
Thanks for your efforts. Overall, the changes look good. If possible, could you post some screenshots of the UI?

File: app/src/main/java/com/starry/myne/ui/theme/Type.kt
Comment (mimi89999) [reply app/src/main/java/com/starry/myne/ui/theme/Type.kt:79]: What about text in italics inside a book? Is that supported?

Comment (starry-shivam) [reply app/src/main/java/com/starry/myne/ui/theme/Type.kt:79]: > What about text in italics inside a book? Is that supported?

No, it currently doesn't take the text formatting of each letter into account. It focuses on extracting text content from HTML/XHTML files inside the EPUB archive and displays it in whatever font the user has selected.

**Comment by mimi89999** [comment]
> Thanks for your efforts. Overall, the changes look good. If possible, could you post some screenshots of the UI?

Sure. Here is one example

File: app/src/main/java/com/starry/myne/ui/theme/Type.kt
Comment (mimi89999) [reply app/src/main/java/com/starry/myne/ui/theme/Type.kt:79]: Indeed italics don't seem to be supported. Maybe we could keep the font anyway, to have it future proof?

Comment (starry-shivam) [reply app/src/main/java/com/starry/myne/ui/theme/Type.kt:79]: > Maybe we could keep the font anyway, to have it future proof?

We can always add them back if or when needed, either from Google Fonts or by going back to this commit and picking up the files. However, I don't think it would be a good idea to increase the APK size by a few megabytes over this for the time being, as it would waste users' bandwidth and potentially their storage.

Personally, I prefer to keep the APK size as small as possible, I hope you understand :)

**Comment by starry-shivam** [comment]
> Sure. Here is one example

Thanks, it looks pretty good, I'll make some UI adjustments (if needed) after merging this successfully,

**Comment by starry-shivam** [comment]
Also, please make sure to update the PR with the base (main) branch by resolving any conflicts, as I've made a few changes to the base branch since this PR was created. If you'd like, I can hold off on adding any more commits until this is merged, so you don't have to resolve conflicts again.

**Review by starry-shivam — APPROVED** [review]
Looks good, thanks!

**Comment by starry-shivam** [comment]
@licaon-kter Just pinging to let you know that the problematic font has been removed.

**Comment by licaon-kter** [comment]
@starry-shivam feel free to ping when the next release is ready so we can remove the AF

**Comment by starry-shivam** [comment]
> @starry-shivam feel free to ping when the next release is ready so we can remove the AF

https://github.com/Pool-Of-Tears/Myne/releases/tag/v4.2.0

**Comment by licaon-kter** [comment]
thanks https://gitlab.com/fdroid/fdroiddata/-/commit/bd4351ead156157a9bfc4a17d1990f8a351151c5

**Pull request merged**
